temaakop/main.js

```diff
@@ -0,0 +1,63 @@
+const list = {}
+let toDo = false
+let inProgress = false
+let done = false
+
+let addTask = (task) =>  list[task] = '';
+
+
+let changeStatus = (task, status) => {
+    if (task in list) {
+        list[task] = status 
+    } else {
+        console.log('Use addTask')
+    }
+}
+
+let deleteTask = (task) => delete list[task];
+
+function showList() {
+
+  console.log(`\n To Do: \n`)
+    for (let key in list) {
+        if (list[key] === 'To Do') {
+          console.log(`   ${key}`)
+          toDo = true
+        } 
+    }
+    if (toDo === false) console.log(` Nothing`)
```

Comment:
Также для бонусной задачи, здесь должен быть прочерк "-"
```suggestion
    if (toDo === false) console.log(`    -`)
```